platform-includes/user-feedback/manual-injection/javascript.nextjs.mdx

```diff
@@ -0,0 +1,26 @@
+```jsx {tabTitle: NextJS}
+function ToggleFeedbackButton() {
+  const [feedback, setFeedback] = useState();
+  // Read `getFeedback` on the client only, to avoid hydration errors when server rendering
```

Comment:
```suggestion
  // Read `getFeedback` on the client only, to avoid hydration errors when server is rendering
```

Reply:
Reading this, I think it's better to call this "to avoid hydration errors during server rendering"

Reply:
Yes, that works!